Assets/Scripts/Checkpoint.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Checkpoint : MonoBehaviour
+{
+    [SerializeField]
+    private float inactivatedRotationSpeed = 100, activatedRotationSpeed = 300;
+
+    [SerializeField]
+    private float inactivatedScale = 1, activatedScale = 1.5f;
+
+    [SerializeField]
+    private Color inactivatedColor, activatedColor;
+
+    private bool isActivated = false;
+    private SpriteRenderer spriteRenderer;
+    private AudioSource audioSource;
+
+    private void Start()
+    {
+        spriteRenderer = GetComponent<SpriteRenderer>();
+        audioSource = GetComponent<AudioSource>();
+        UpdateColor();
+        
```

Comment:
I'd try to leave as little space as possible. Maybe space between lines of code, but limit any after that.

Reply:
All right, I deleted some space.